### PR TITLE
Make lightgbm examples more practical

### PR DESCRIPTION
**Pull request by smly**
Tuning `boosting_type` and `drop_rate` is helpful to understand the usage/advantage of define-by-run. However, practically, these parameters are not important for accuracy. This PR makes LightGBM examples more practical.

### CODE REVIEW

File: examples/lightgbm_simple.py [hunk 1]
```diff
@@ -37,18 +40,16 @@ def objective(trial):
...
-        'num_leaves': trial.suggest_int('num_leaves', 10, 1000),
-        'learning_rate': trial.suggest_loguniform('learning_rate', 1e-8, 1.0)
+        'boosting_type': 'gbdt',
+        'lambda_l1': trial.suggest_loguniform('lambda_l1', 1e-8, 10.0),
+        'lambda_l2': trial.suggest_loguniform('lambda_l2', 1e-8, 10.0),
+        'num_leaves': trial.suggest_int('num_leaves', 2, 256),
+        'feature_fraction': min(trial.suggest_uniform('feature_fraction', 0.4, 1.0 + EPS), 1.0),
+        'bagging_fraction': min(trial.suggest_uniform('bagging_fraction', 0.4, 1.0 + EPS), 1.0),
+        'bagging_freq': trial.suggest_int('bagging_freq', 1, 7),
+        'min_child_samples': int(trial.suggest_uniform('min_child_samples', 5, 100 + EPS)),
```
Comment (sile) [inline examples/lightgbm_simple.py]: How about using `suggest_int` as follows?
```suggestion
        'min_child_samples': trial.suggest_int('min_child_samples', 5, 100),
```

File: examples/pruning/lightgbm_integration.py [hunk 2]
```diff
@@ -30,18 +33,16 @@ def objective(trial):
...
-        'num_leaves': trial.suggest_int('num_leaves', 10, 1000),
-        'learning_rate': trial.suggest_loguniform('learning_rate', 1e-8, 1.0)
+        'boosting_type': 'gbdt',
+        'lambda_l1': trial.suggest_loguniform('lambda_l1', 1e-8, 10.0),
+        'lambda_l2': trial.suggest_loguniform('lambda_l2', 1e-8, 10.0),
+        'num_leaves': trial.suggest_int('num_leaves', 2, 256),
+        'feature_fraction': min(trial.suggest_uniform('feature_fraction', 0.4, 1.0 + EPS), 1.0),
+        'bagging_fraction': min(trial.suggest_uniform('bagging_fraction', 0.4, 1.0 + EPS), 1.0),
+        'bagging_freq': trial.suggest_int('bagging_freq', 1, 7),
+        'min_child_samples': int(trial.suggest_uniform('min_child_samples', 5, 100 + EPS)),
```
Comment (sile) [inline examples/pruning/lightgbm_integration.py]: ditto

File: examples/lightgbm_simple.py (hunk 1 above)
Comment (smly) [reply examples/lightgbm_simple.py]: Good! I'll fix it.

File: examples/pruning/lightgbm_integration.py (hunk 2 above)
Comment (smly) [reply examples/pruning/lightgbm_integration.py]: Fixed: d22a7ce

File: examples/lightgbm_simple.py (hunk 1 above)
Comment (smly) [reply examples/lightgbm_simple.py]: Fixed: d22a7ce
